### PR TITLE
Add support for ArgMax and ArgMin in C2 onnx backend and frontend

### PR DESCRIPTION
**Pull request by houseroad**
Pass the end to end test cases in https://github.com/onnx/onnx/pull/1049

### CODE REVIEW

File: caffe2/onnx/onnx_exporter.cc [hunk 1]
```diff
@@ -351,6 +353,32 @@ ConvertedResult OnnxExporter::CommonCaffe2OpToOnnxNodes(
...
+    const caffe2::OperatorDef& def,
+    const std::unordered_map<std::string, caffe2::TensorShape>& shapes) {
+  auto result = CommonCaffe2OpToOnnxNodes(def);
+  auto& nodes = result.first;
+
+  CAFFE_ENFORCE_EQ(nodes.size(), 1);
+  auto& node = nodes.back();
+
+  bool explicit_axis = false;
+  for (const auto& a : def.arg()) {
```
Comment (bddppq) [inline caffe2/onnx/onnx_exporter.cc]: `ArgumentHelper::HasArgument`

**Review by bddppq — APPROVED** [review]

File: caffe2/onnx/onnx_exporter.cc (hunk 1 above)
Comment (houseroad) [reply caffe2/onnx/onnx_exporter.cc]: Right, I forgot my cute function.

**Review by Ac2zoom — APPROVED** [review]
Everything looks good to me!  Implementation matches that of similar ops and manifests expected divergences in ONNX vs. Caffe2 specs.

**Pull request closed without merging**